examples/advanced_circuits_algorithms/Grover/Grover.ipynb

```diff
@@ -133,11 +133,6 @@
    ]
```

Comment:
<p>Same, will need to re-run all cells. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/amazon-braket/amazon-braket-examples/pull/619/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>